src/Microsoft.AspNetCore.Razor.Language/Resources.resx

```diff
@@ -533,4 +533,7 @@ Instead, wrap the contents of the block in "{{}}":
   <data name="PropertyMustNotBeNull" xml:space="preserve">
     <value>The '{0}.{1}' property must not be null.</value>
   </data>
+  <data name="MissingFeatureDependency" xml:space="preserve">
+    <value>The '{0}' is missing feature '{1}'.</value>
+  </data>
```

Comment:
This could use some love. Look at the error messages for RazorEngine and get inspired.

Reply:
lol did. They're nearly identical 😉. I'll change the Resource name to be razor project engine specific. I think you're misunderstanding this error for a feature missing another feature dependency.

Reply:
Actually, going to change the direction on this. Not going to make the `IRazorImportFeature` a required feature. If the feature isn't available we'll assume 0 imports.